app/assets/stylesheets/earthworks.css

```diff
@@ -120,6 +121,12 @@
   border: 1px solid #ABABA9;
   padding: .5rem;
   border-radius: .25rem;
+  background-color: white;
```

Comment:
Can you explain why this is needed?  It already appears white to me.

Reply:
Without the background set. Because we aren't using the input border and putting our own on it needs a background color because without it inherits from the navbar-search css. 
![Screenshot 2024-08-21 at 12 49 56 PM](https://github.com/user-attachments/assets/76b3c36f-45c7-41e8-aad4-e55cbf6cf180)

Reply:
I see a new version of geoblacklight.

Can you instead set `--bs-bg-opacity: 0`?

Reply:
Doesn't work. Neither does --bs-navbar-color: white.

Reply:
I've pulled this branch and I still see that background.

Reply:
Ah, I see this is just inside the input-group.  Is there a `.search-query-form` that is not within the `.navbar-search`.  I can only see this being important in the latter case right now.